jazzy/distribution.yaml

```diff
@@ -3560,6 +3560,12 @@ repositories:
       url: https://github.com/ros-perception/laser_proc.git
       version: ros2-devel
     status: maintained
+  laser_segmentation:
+    doc:
+      type: git
+      url: https://github.com/ajtudela/laser_segmentation.git
+      version: jazzy
```

Comment:
Please choose a branch that exists, or create the `jazzy` branch on this repo

```
ERROR: In file '''/home/runner/work/rosdistro/rosdistro/jazzy/distribution.yaml''': Could not validate doc entry for repo sicks300_ros2 with error [[[Could not validate repository with url https://github.com/ajtudela/sicks300_ros2.git and version jazzy from entry at line 8568. Error reason: No branch found matching jazzy]]]
```

Reply:
That branch exists: https://github.com/ajtudela/laser_segmentation/tree/jazzy